src/views/UserSearch/Filters.js

```diff
@@ -161,6 +185,25 @@ class Filters extends React.Component {
             aria-labelledby="users-filter-accordion-tags"
           />
         </Accordion>
+        {
+          isConsortium && (
+            <Accordion
+              displayClearButton
+              id={`${ACCORDION_ID_PREFIX}_userTypes`}
```

Comment:
`-user-types`. Follow the established convention in HTML IDs of separating words with hyphens, e.g. `patron-group` not `patronGroup`.